src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -432,10 +433,12 @@ protected function dateBasedWhere($type, Builder $query, $where)
      */
     protected function wrapValue($value)
     {
-        if ($this->isReserved($value)) {
-            return parent::wrapValue($value);
+        if ($value === '*') {
+            return $value;
         }
 
-        return $value !== '*' ? sprintf($this->wrapper, $value) : $value;
+        $value = $this->isReserved($value) ? Str::lower($value) : Str::upper($value);
+
+        return '"' . str_replace('"', '""', $value) . '"';
```

Comment:
You got a point there. I just re-implemented the original laravel code and this works well on my app including the tests. Any recommendation on how to handle this?